docs/en/pid-eaa-issuance.rst

```diff
@@ -192,13 +233,13 @@ The JWS payload of the request object is represented below:
     }
 
 
-**Steps 16-18:** The Wallet Instance creates a new key pair to which the new credential shall be bound. Then, it creates a proof of possession with the new key and the *c_nonce* obtained in **Step 15** and it creates a DPoP proof for the request to the PID credential issuance endpoint.  
+**Steps 16-18 (DPoP Proof for Credential Endpoint):** The Wallet Instance creates a new key pair to which the new credential SHALL be bound. Then, it creates a proof of possession with the new key and the *c_nonce* obtained in **Step 15** and it creates a DPoP proof for the request to the PID/(Q)EAA credential issuance endpoint.  
```

Comment:
How about using the same previously generated key for `DPoP` to bind to the new credential? Since both operations take place within seconds of each other, in my opinion, there is no need to generate a new key pair.

Reply:
In principle, using the same key breaks the key separation principle of using different keys for different purposes. However in this case, I don't see any security issues. I will change in SHOULD as suggested by @peppelinux .

Reply:
Thanks!